happi/cli.py

```diff
@@ -59,18 +71,10 @@ def happi_cli(args):
     if args.cmd == 'search':
         logger.debug("We're in the search block")
 
-        # Ensure we have an even number of search elements
-        # Should always have key:value pairs
-        if len(args.search_criteria) % 2 != 0:
-            raise SearchError('Search criteria should be given as key '
-                              'value pair\ni.e:\n'
-                              'happi search beamline MFX stand DG1')
-
         # Get search criteria into dictionary for use by client
         client_args = {}
-        for i in range(0, len(args.search_criteria), 2):
-            criteria = args.search_criteria[i]
-            value = args.search_criteria[i+1]
+        for user_arg in args.search_criteria:
+            criteria, value = user_arg.split('=')
```

Comment:
```suggestion
            criteria, value = user_arg.split('=', 1)
```

Reply:
To clarify, is this for inputs like `name=Ilikethe=sign`?

Reply:
Yes. However unlikely it is that there will be `=` in `value`, it's easier to just build this in now.